configurations/gcp/user-agent-config.yml

```diff
@@ -0,0 +1,4 @@
+---
+- type: replace
+  path: /cloud_provider/properties/google/user_agent_prefix?
+  value: "kubo/0.0.3"
```

Comment:
I think its appropriate for now. At some point we'll want to lock versions of kubo-deployment to kubo-release and this repo will become aware of the version, until then!